funcx_endpoint/funcx_endpoint/executors/high_throughput/executor.py

```diff
@@ -858,7 +858,12 @@ def shutdown(self, hub=True, targets="all", block=False):
         # self.outgoing_q.close()
         # self.incoming_q.close()
         if self.queue_proc:
-            self.queue_proc.terminate()
+            try:
+                self.queue_proc.terminate()
+            except AttributeError:
+                log.info("Executor interchange terminate skipped due to wrong context")
```

Comment:
`AttributeError` handling? And turning it into `info`?
Are you certain this is what you want here. I would just let such an error raise, or at least reraise after any logging.